website/docs/usage.md

```diff
@@ -25,6 +25,40 @@ const App = () => {
 };
 ```
 
+## Custom Columns and Rows
+
+The Spreadsheet component requires the `columnLabels` prop or the `rowLabels` prop which accepts array. If no `columnLabels` is passed, it defaults to alphabetical labels and if no `rowLabels` are passed it defaults to row index labels.
```

Comment:
I think requires is not the right wording as these are optional parameters. How about "accepts" instead?